frontend/src/components/Header/components/AuthButtons/index.tsx

```diff
@@ -80,16 +93,24 @@ function LoggedInButtons({ name, email, logout }: { name?: string; email?: strin
             <MenuDivider />
           </div>
         )}
-        <MenuItem data-testid="log-out" onClick={logout}>
+        <MenuItem data-testid="log-out" onClick={handleLogout}>
           <LogOutText>Logout</LogOutText>
           <LogOutEmail data-testid="user-email">{email}</LogOutEmail>
         </MenuItem>
       </Menu>
     );
   }
+
+  function handleLogout() {
```

Comment:
this is needed because logout() expects its argument to be options, not click event